src/commands/cvms/create.ts

```diff
@@ -14,257 +14,250 @@ import { parseEnv } from '@/src/utils/secrets';
+      teepodsSpinner.stop(true);
+      if (teepods.nodes.length === 0) {
+        logger.error('No TEEPods available. Please try again later.');
+        process.exit(1);
+      }
 
-  if (options.debug) {
-    logger.debug('Public key:', pubkey.app_env_encrypt_pubkey);
-    logger.debug('Encrypted environment variables:', encrypted_env);
-    logger.debug('Environment variables:', JSON.stringify(envs));
-  }
+      let selectedTeepod: TEEPod;
+      // Fetch available TEEPods
+      if (!options.teepodId) {
+        selectedTeepod = teepods.nodes[0];
+        if (!selectedTeepod) {
+          logger.error('Failed to find default TEEPod');
+          process.exit(1);
+        }
+      } else {
+        selectedTeepod = teepods.nodes.find(pod => pod.teepod_id === Number(options.teepodId));
+        if (!selectedTeepod) {
+          logger.error('Failed to find selected TEEPod');
+          process.exit(1);
+        }
+      }
 
-  const createSpinner = logger.startSpinner('Provisioning CVM');
-  const response = await createCvm({ ...vmConfig, encrypted_env, app_env_encrypt_pubkey: pubkey.app_env_encrypt_pubkey, app_id_salt: pubkey.app_id_salt });
-  createSpinner.stop(true);
-  if (!response) throw new Error('Failed to create CVM');
+      let selectedImage: Image;
+      if (!options.image) {
+        selectedImage = selectedTeepod.images?.find(image => image.name === DEFAULT_IMAGE);
+        if (!selectedImage) {
+          logger.error(`Failed to find default image ${DEFAULT_IMAGE}`);
+          process.exit(1);
+        }
+      } else {
+        selectedImage = selectedTeepod.images?.find(image => image.name === options.image);
+        if (!selectedImage) {
+          logger.error(`Failed to find selected image: ${options.image}`);
+          process.exit(1);
+        }
+      }
 
-  logger.success('CVM created successfully');
-  logger.break();
-  const tableData: { [key: string]: any } = {
-    'CVM ID': response.vm_uuid.replace(/-/g, ''),
-    'App ID': response.app_id,
-    'Name': response.name,
-    'Status': response.status,
-    'Endpoint': `${CLOUD_URL}/dashboard/cvms/${response.vm_uuid.replace(/-/g, '')}`,
-    'Created At': new Date(response.created_at).toLocaleString(),
-  };
-  if (response.kms_contract_address) tableData['KMS Contract Address'] = response.kms_contract_address;
-  if (response.kms_owner_address) tableData['KMS Owner Address'] = response.kms_owner_address;
-  logger.keyValueTable(tableData);
-}
+      // Prepare VM configuration
+      const vmConfig = {
+        teepod_id: selectedTeepod.teepod_id,
+        name: options.name,
+        image: selectedImage.name,
+        vcpu: vcpu,
+        memory: memory,
+        disk_size: diskSize,
+        compose_manifest: {
+          docker_compose_file: composeString,
+          docker_config: {
+            url: '',
+            username: '',
+            password: '',
+          },
+          features: ['kms', 'tproxy-net'],
```

Comment:
these lines can be removed.

Reply:
`docker_config` and `features` are both unnecessary? just make sure because this is the original implementation

Reply:
Yes, because:
1. Private image registry support has been moved to pre-launch scripts, and the built-in support is now incomplete.
2. The `features` field is for backward compatibility and has already been removed in Dstack 0.5.x. The Phala Cloud backend will automatically patch the app compose file, so the one from users should be as minimal as possible.